File: DSA Practice/LeetCode/997. Find the Town Judge/solution.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        index = [0] * n
        for i, j in trust:
            index[i-1] = -1
            index[j-1] += 1
        #print(index)
        if (n-1) in index:
            return index.index(n-1) + 1
        else :
            return -1
            

File: DSA Practice/LeetCode/997. Find the Town Judge/solution.java
class Solution {
    public int findJudge(int n, int[][] trust) {
        int[] people = new int[n];

        for (int[] i : trust){
            people[i[0]-1] = -1;
            people[i[1]-1]++;
        }

        //System.out.println(Arrays.toString(people));

        int res = -1;

        for (int i = 0; i < n; i++){
            if (people[i] == n - 1){
                res = i + 1;
                break;
            }
        }

        return res;
    }
}